client/state/upgrade.go

```diff
@@ -82,15 +82,14 @@ func backupDB(bdb *bbolt.DB, dst string) error {
 
 // UpgradeAllocs upgrades the boltdb schema. Example 0.8 schema:
 //
-//	* allocations
-//	  * 15d83e8a-74a2-b4da-3f17-ed5c12895ea8
-//	    * echo
-//	       - simple-all (342 bytes)
-//	     - alloc (2827 bytes)
-//	     - alloc-dir (166 bytes)
-//	     - immutable (15 bytes)
-//	     - mutable (1294 bytes)
-//
+//   - allocations
+//   - 15d83e8a-74a2-b4da-3f17-ed5c12895ea8
+//   - echo
+//   - simple-all (342 bytes)
+//   - alloc (2827 bytes)
+//   - alloc-dir (166 bytes)
+//   - immutable (15 bytes)
+//   - mutable (1294 bytes)
```

Comment:
I think the indentation here is relevant? Is that a bug in `gofmt`?

Reply:
Good catch, I think we can just eliminate the dashes so gofmt doesn't assume this is a list